gtfs/gtfs_utils/gtfs_utils/s3.py

```diff
@@ -63,6 +63,14 @@ def get_latest_file(crud: S3Crud,
                     mot_file_name: str,
                     desired_date: Union[datetime.datetime, datetime.date],
                     past_days_to_try: int = 100) -> Tuple[datetime.date, str]:
+    """
+    for mot file type return it's newest version before `desired_date`
+    :param crud: S3Crud object
+    :param mot_file_name: Original name of the file from MOT
+    :param desired_date: the newest date acceptable
+    :param past_days_to_try: how back to search for the file
```

Comment:
"how back to search for the file" -> "how many days to search backwards for the file"